src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -279,6 +286,37 @@ public function addClassResource(\ReflectionClass $class)
     }
 
     /**
+     * Checks whether the requested class, interface or trait exists and registers the result for resource tracking.
+     *
+     * @param string $class
+     *
+     * @return bool
+     *
+     * @final
```

Comment:
in fact, we already have one: proxies are not loaded via standard autoloading, so they don't get through DebugClassLoader :)